packages/atlas/src/views/viewer/CategoryView/CategoryView.tsx

```diff
@@ -28,25 +31,22 @@ export const CategoryView = () => {
   const mdBreakpointMatch = useMediaMatch('md')
   const { id = '' } = useParams()
 
-  const { categories, totalVideosCount, loading } = useCategories()
+  const { totalVideosCount, loading } = useQnCategories()
```

Comment:
This `totalVideosCount` won't be the correct value for us - it will be total count of all videos, but what we should be interested in is the total count of videos that could be displayed in the app - so taking into account only whitelisted categories. I think we should remove `totalVideosCount` from `useQnCategories` (My guess is that once you implement changes to video workspace, `useQnCategories` won't be used anywhere in the app, outside some hooks). Instead, you could add `totalVideosCount` to `useVideoCategoriesWithCounter`